src/event_handler.py

```diff
@@ -7,37 +7,44 @@
 from urllib.request import Request, urlopen
 from urllib.error import URLError, HTTPError
 
-
-# The base-64 encoded, encrypted key (CiphertextBlob) stored in the ENCRYPTED_SLACK_WEBHOOK_URL environment variable
+# KMS-encrypted ciphertext. Note that the ciphertext has been base64-encoded, in order for the
+# ciphertext to be represented as string (KMS CLI does this automatically when specifying output
+# as text). It will, therefore, need to be decoded first before decrypting (like done below).
 ENCRYPTED_HOOK_URL = os.environ['ENCRYPTED_SLACK_WEBHOOK_URL']
 SLACK_CHANNEL = os.environ['SLACK_CHANNEL']
 
-HOOK_URL = "https://" + boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_HOOK_URL))['Plaintext'].decode('utf-8')
+decrypted_hook_url = boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_HOOK_URL))
+HOOK_URL = "https://" + decrypted_hook_url['Plaintext'].decode('utf-8')
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 
+def build_slack_message(message):
+    alarm_name = message['AlarmName']
+    old_state = message['OldStateValue']
+    new_state = message['NewStateValue']
+    reason = message['NewStateReason']
+
+    slack_message = f"CloudWatch Alarm state changed for `{alarm_name}`"
+    slack_message += f" from `{old_state}` to `{new_state}`"
+    slack_message += f" because of the following reason:\n```{reason}```"
```

Comment:
Could use `\` here:
```
slack_message = f"CloudWatch Alarm state changed for `{alarm_name}`" \
                f" from `{old_state}` to `{new_state}`" \
                f" because of the following reason:\n```{reason}```"
```

Reply:
That does looks cleaner 👍